docs/AboutUs.md

```diff
@@ -7,17 +7,17 @@ We are a team based in the [School of Computing, National University of Singapor
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-## Project team
+## CodeConnect
 
-### John Doe
+### Song Ivan
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Songivan.png" width="200px">
```

Comment:
The filename of the profile photo should be docs/images/github_username_in_lower_case.png
Note the need for lower case
https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html

Reply:
okay will change thank you